### PR TITLE
Fixes pod survivors not being able to customise their name on spawn in

### PR DESCRIPTION
**Pull request by Jaraci**
See title.

### CODE REVIEW

**Comment by Jaraci** [comment]
!review

File: html/changelogs/omicega-renamebugfix.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixes ghost spawner pod survivors not being allowed to pick a name in addition to customising their appearance.
```
Comment (WickedCybs) [inline html/changelogs/omicega-renamebugfix.yml:41]:
```suggestion
  - bugfix: "Fixes ghost spawner pod survivors not being allowed to pick a name in addition to customising their appearance."
```

Comment (WickedCybs) [reply html/changelogs/omicega-renamebugfix.yml:41]: Was just missing a quotation mark, here. Should pass the CI with it fixed.

Comment (Jaraci) [reply html/changelogs/omicega-renamebugfix.yml]: oopsie

**Review by Geevies — APPROVED** [review]

**Review by Doxxmedearly — APPROVED** [review]

**Pull request merged**